config/jobs/gardener/gardener-apidiff.yaml

```
@@ -0,0 +1,17 @@
+presubmits:
+  gardener/gardener:
+  - name: pull-gardener-apidiff
+    cluster: gardener-prow-build
+    skip_if_only_changed: "^docs/|\\.md$"
+    decorate: true
+    optional: true
+    branches:
+    - ^master$ # run on master branch
```

Can you add the release branches?
```suggestion
    - ^master$
    - ^release-v\d+.\d+$
```

---

Right, I forgot about that.